services/fluent-bit/0.19.21/kustomization.yaml

```diff
@@ -2,3 +2,4 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
   - fluent-bit.yaml
+  - grafana-dashboards-logging
```

Comment:
![image](https://user-images.githubusercontent.com/5897740/184431755-a8dba3f3-66bc-4450-9142-3a2fc2e3603c.png)
After manually pushing these changes to a cluster, the audit dashboard disappeared (since fluent-bit was not installed) (ignore the other dashboards, those were created ad hoc)